BsddRevitPlugin.Common/Commands/IFCexporter.cs

```diff
@@ -56,358 +58,21 @@ public class IFCexporter : IExternalCommand
                 //// Call the OnIFCExport methods
                 //ifcCommandOverrideApplication.OnIFCExport(uiApp, commandEvent);
 
+                using (Transaction transaction = new Transaction(document, "bSDD IFC export"))
+                {
+                    List<IFCClassification> bsddClassifications = IfcClassificationManager.GetAllIfcClassificationsInProject();
+                    List<IFCClassification> storedClassifications = IfcClassificationManager.GetStoredClassifications(document);
 
+                    // Only restore the first item in the list, like revit-ifc does
+                    storedClassifications = storedClassifications.Any() ? new List<IFCClassification> { storedClassifications[0] } : new List<IFCClassification>();
 
-                using (Transaction transaction = new Transaction(doc, "Export IFC"))
-                {
+                    // Store temporary bSDD Classifications
+                    IfcClassificationManager.UpdateClassifications(new Transaction(document, "Store temporary bSDD Classifications"), document, bsddClassifications, IfcClassificationManager.getBsddClassificationParameterMap());
```

Comment:
Storing the old classification doesn't work.

Reply:
Even though the settings are reset correctly, it still exports Uniformat:
![image](https://github.com/buildingsmart-community/bSDD-Revit-plugin/assets/100413270/dbdee614-0fe5-48fd-bf5e-3f5e93b70186)

Reply:
It has something to do with the Classification Field Name that doesn't get set in the resetted classification entity